AnkiDroid/src/main/java/com/ichi2/anki/ModelFieldEditor.java

```diff
@@ -441,6 +441,24 @@ private void sortByField() {
         }
     }
 
+    /*
+     * Toggle the "Remember last input" setting AKA the "Sticky" setting
+     */
+    private void toggleStickyField() {
+        try {
+            // Get the current field
+            JSONObject field = (JSONObject) mNoteFields.get(mCurrentPos);
+            // If the sticky setting is enabled then disable it, otherwise enable it
+            if (field.getBoolean("sticky")){
+                field.put("sticky", false);
+            } else {
+                field.put("sticky", true);
+            }
+        } catch (JSONException e) {
+            throw new RuntimeException(e);
+        }
```

Comment:
Hey @sudomain ! Thanks for your patience on this. We are very close to working through the PR backlog so I finally looked at this again :-). I'm prepared to merge it as is, the only change I suggest is that I don't think we need the try/catch boilerplate around JSON access now, after a change from Arthur Milchior in #5550 - so this suggestion removes that. I'll try committing it to make sure this still passes CI checks and if it does then I'll merge

```suggestion
        // Get the current field
        JSONObject field = (JSONObject) mNoteFields.get(mCurrentPos);
        // If the sticky setting is enabled then disable it, otherwise enable it
        if (field.getBoolean("sticky")) {
            field.put("sticky", false);
        } else {
            field.put("sticky", true);
        }
```